amqp/subscriber.go

```diff
@@ -28,7 +30,7 @@ type Subscriber interface {
 	SubscribeDeviceUplink(appID, devID string, handler UplinkHandler) error
 	SubscribeAppUplink(appID string, handler UplinkHandler) error
 	SubscribeUplink(handler UplinkHandler) error
-	ConsumeUplink(queue string, handler UplinkHandler) error
+	ConsumeMessages(queue string, uplinkHandler UplinkHandler, eventHandler DeviceEventHandler) error
```

Comment:
OK yeah, I see the need for one queue. So when creating and binding the queue, in the controller, the right topic(s) are chosen. The process, indeed, needs to have a generic consumer. Integrations that are not interested in device events would pass a nil handler for events, in fact there wouldn't be events bound to the queue